src/client/components/pages/entities/author.js

```diff
@@ -81,6 +81,11 @@ function AuthorAttributes({author, averageRating}) {
 								stars={5}
 							/>
 						</dd>
+						{reviewsCount ?
```

Comment:
I think it would be good to show "No reviews" or "0 reviews" when reviewsCount=0.
Currently we are not rendering anything i reviewsCount=0.

Reply:
That's a good idea. I've made the changes